frontend/app/components/public-schools/existing-conditions/recently-built-schools.js

```diff
@@ -1,6 +1,18 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 
 export default Component.extend({
+  // ceqr_school_buildings dataset is a combination of two datasets lcgms and bluebook
+  // lcgms dataset represents schools that opened recently
+  newlyOpenedSchools: computed('analysis.ceqr_school_buildings', function() {
```

Comment:
@trbmcginnis since this is the same function defined in [existing-conditions-steps.js](frontend/app/components/public-schools/existing-conditions/existing-conditions-steps.js), I wonder if there's a way to share this code between the the places it's needed?

Reply:
That's a great idea -- I'm going to go ahead and add `newlyOpenedSchools` to the model as a computed property